pkg/util/deployment/deployment.go

```diff
@@ -53,9 +55,12 @@ func GetOldReplicaSets(deployment extensions.Deployment, c clientset.Interface)
 		})
 }
 
+type rsListFunc func(string, api.ListOptions) ([]extensions.ReplicaSet, error)
```

Comment:
Nevermind. Add a TODO here about investigating to switching to full namespacers and I can look into it if I ever find time.